src/schema/v2/requestLocation.ts

```diff
@@ -34,14 +39,16 @@ export const RequestLocationField: GraphQLFieldConfig<void, ResolverContext> = {
     },
   },
   resolve: async (_root, args, { requestLocationLoader }) => {
-    const { body, headers } = await requestLocationLoader({ ip: args.ip })
+    const { body, headers, cached } = await requestLocationLoader({
```

Comment:
I only know it was there because I had initially included it on all requests to ease debugging when I first built MP :)

https://github.com/artsy/metaphysics/blob/048107b32ec677e7922422f84d6dab74de00d49a/src/lib/cache.ts#L143-L149